webapp/webapp.v3.spec.yaml

```diff
@@ -504,6 +504,10 @@ components:
         latest_only:
           type: boolean
           enum: [true, false]
+        third_party:
```

Comment:
Does "third_party" flag in requests take any effect? I tried errata, cve, packages endpoints without third_party flag set and also set to false and for all I got data also from third party repositories what's wrong I guess.